polylogue/cli_common.py

```
@@ -172,3 +175,65 @@ def should_download(local_path: Path, remote_modified_time: Optional[str], *, fo
+        The selection if valid, None if cancelled/empty
+    """
+    if selection is None:
+        console.print(f"[yellow]{action.title()} cancelled; no {item_type} selected.")
+        return None
+    if not selection:
+        console.print(f"[yellow]No {item_type} selected; nothing to {action}.")
+        return None
+    return selection
+
+
+def resolve_inputs(path: Path, plain: bool) -> Optional[List[Path]]:
```

Ruff (0.15.9)

[warning] Boolean-typed positional argument in function definition

(FBT001)